src/NServiceBus.RabbitMQ/Routing/ConventionalRoutingTopology.cs

```diff
@@ -79,6 +79,11 @@ public void DeclareAndInitialize(IModel channel, IEnumerable<string> receivingAd
                 channel.QueueDeclare(address, useDurableExchanges, false, false, null);
                 Initialize(channel, address);
             }
+
+            foreach (var address in receivingAddresses)
+            {
+                channel.ExchangeBind(address, "nsb.delay-destination", $"#.{address}");
```

Comment:
This blows up our Unit Tests. As this method is being created in here:
https://github.com/Particular/NServiceBus.RabbitMQ/blob/native-delays/src/NServiceBus.RabbitMQ.Tests/Support/ConventionalRoutingTopologyExtensions.cs#L23
And then ExchangeBind fails, as at that point this exchange doesn't exist.